schema.graphql

```diff
@@ -16,3 +16,97 @@ type GaugeStakingHistory @entity {
   lastBlockNumber_: BigInt!
   lastBlockTimestamp_: BigInt!
 }
+
+type Builder @entity(immutable: false) {
+  id: Bytes!
+  gauge_: Bytes!
+  backerRewardPercentage_: BigInt!
+  rewardReceiver_: Bytes!
+  state_: BuilderState @derivedFrom(field: "builder_")
+  isHalted_: Boolean!
+  lastCycleRewards_: BigInt!
+  totalAllocation_: BigInt!
+  rewardShares_: BigInt!
```

Comment:
Not managed at the moment. Do we need it?

Reply:
I updated but right now we are querying the contract, maybe we can introduce an event